### PR TITLE
Fix #609: Fix presence of ID value in record edit forms

### LINKED ISSUE #609: If id field has custom title/description, its value is not loaded
screenshot from 2018-09-26 11-16-08

screenshot_2018-09-26 kinto web administration

### PR DESCRIPTION
**Pull request by leplatrem**
fixes #609

### CODE REVIEW

**Comment by leplatrem** [comment]
screenshot from 2018-09-26 11-24-10

File: test/components/RecordAttributes_test.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    it("should show the id as disabled", () => {
+      const field = node.querySelector("#root_id");
+      expect(field.value).to.eql("abc");
+      expect(field.hasAttribute("disabled")).to.be.true;
+    });
+
+    it("should show the id as disabled", () => {
+      const field = node.querySelector("#root_foo");
+      expect(field.value).to.eql("bar");
+    });
```
Comment (glasserc) [inline test/components/RecordAttributes_test.js]: What's this test about? The name doesn't seem to match.

**Comment by leplatrem** [comment]
> what's the intended behavior for the id field?

For some use cases, it makes sense to be able to specify the ID field while creating a record. For some others, the notion of record ID can be completely hidden.

File: src/components/record/RecordForm.js [hunk 2]
```diff
@@ -45,6 +45,20 @@ export function extendSchemaWithAttachment(
...
 
+export function extendUIWithKintoFields(
+  uiSchema: Object,
+  isCreate: boolean
+): Object {
+  return {
+    ...uiSchema,
+    id: {
+      "ui:widget": "text",
+      "ui:disabled": !isCreate,
```
Comment (glasserc) [inline src/components/record/RecordForm.js]: Should this maybe be `ui:readonly` instead of disabled?

Comment (leplatrem) [reply src/components/record/RecordForm.js]: Good question...

We don't need the `id` value to get passed with the form data.

Should the id field be focusable? included in the tabbed field navigation?

Comment (glasserc) [reply src/components/record/RecordForm.js]: I just thought a user might want to copy-paste the ID. I guess they can already grab it from the breadcrumbs, though, so maybe not a big deal?

Comment (leplatrem) [reply src/components/record/RecordForm.js]: I would leave it disabled :)

**Pull request merged**